example/styles.css

```diff
@@ -160,9 +160,6 @@ button:not(.cf-btn) {
 }
 
 .cf-card__footer_message {
```

Comment:
@wyuenho Should we need to consider if someone else using `<Text>` inside `<CardContent>`?

Example:
```
<CardContent title="xxxx" footerMessage="xxx">
       <Text case="titlecase">hello world</Text>
</CardContent>
// or
<CardContent title="xxxx" footerMessage="xxx">
       <Text type="error">Hello World</Text> 
</CardContent>
```